modules/sandboxed-containers-scheduling-workloads.adoc

```diff
@@ -3,5 +3,60 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-scheduling-workloads_{context}"]
+= Scheduling {sandboxed-containers-first} workloads
 
-= Scheduling sandboxed containers workloads
+You can schedule your workloads to run on {sandboxed-containers-first}.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
```

Comment:
Should you specify which objects take `spec.runtimeClassName: kata` and which take `spec.template.spec.runtimeClassName: kata`? Or would that be obvious to users?

Reply:
This is something that we can consider post release.